src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoServiceSpec.scala

```diff
@@ -87,10 +87,10 @@ class LeonardoServiceSpec extends TestKit(ActorSystem("leonardotest")) with Flat
     gdDAO.firewallRules(project).name.value shouldBe proxyConfig.firewallRuleName
 
     // should have created init and staging buckets
-    val initBucketOpt = storageDAO.buckets.keys.find(_.value.startsWith(name1.value+"-init"))
+    val initBucketOpt = storageDAO.buckets.keys.find(_.value.startsWith("leoinit-"+name1.value))
     initBucketOpt shouldBe 'defined
 
-    val stagingBucketOpt = storageDAO.buckets.keys.find(_.value.startsWith(name1.value+"-staging"))
+    val stagingBucketOpt = storageDAO.buckets.keys.find(_.value.startsWith("leoinit-"+name1.value))
```

Comment:
Should the staging bucket name be `"leostaging-"+name1.value`

Reply:
+1, good catch

Reply:
oh man, and tests didn't catch this :(